packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -243,6 +244,24 @@ export type TableProps<
    * @default false
    */
   enableColumnReordering?: boolean;
+  /**
+   * Function that returns index of the row that you want to scroll to.
+   *
+   * It doesn't work with paginated tables.
+   * @example
+   * <Table
+   *   data={data}
+   *   scrollToRow={(rows) => rows.findIndex((row) => row.original === data[250])}
+   *   {...restProps}
+   * />
+   * @example
+   * <Table
+   *   scrollToRow={(rows) => rows.findIndex((row) => row.id === item.id)}
```

Comment:
Maybe enough the first example? here is not even clear, what is that `item`

Reply:
Just now realized that I could write this without using `getRowId`:
`scrollToRow={(rows) => rows.findIndex((row) => row.original.id === data[12345].id)}`